hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -605,7 +607,9 @@ public void createDirectory(final Path path, final FsPermission permission, fina
     }
   }
 
-  public AbfsInputStream openFileForRead(final Path path, final FileSystem.Statistics statistics)
+  public AbfsInputStream openFileForRead(final Path path,
```

Comment:
Can we method overloading like 
`  public AbfsInputStream openFileForRead(final Path path,
                                         final FileSystem.Statistics statistics)
          throws AzureBlobFileSystemException {
   return openFileForRead(path,Optional.empty(), statistics);
  }` so as to avoid changing everywhere in the tests especially.